src/common/hooks/useInstance.js

```diff
@@ -3,18 +3,23 @@ import {
   useMemo,
 } from 'react';
 
+import { useStripes } from '@folio/stripes-core';
```

Comment:
Please import from `@folio/stripes/core` to use `stripes` peerDep

Reply:
automatic import got me there